scripts/brp-strip

```diff
@@ -5,16 +5,13 @@ if [ -z "$RPM_BUILD_ROOT" -o "$RPM_BUILD_ROOT" = "/" ]; then
 fi
 
 STRIP=${1:-strip}
+NCPUS=${RPM_BUILD_NCPUS:-1}
 
 case `uname -a` in
 Darwin*) exit 0 ;;
 *) ;;
 esac
 
 # Strip ELF binaries
-for f in `find "$RPM_BUILD_ROOT" -type f \( -perm -0100 -or -perm -0010 -or -perm -0001 \) -exec file {} \; | \
-        grep -v "^${RPM_BUILD_ROOT}/\?usr/lib/debug"  | \
-	grep -v ' shared object,' | \
-	sed -n -e 's/^\(.*\):[ 	]*ELF.*, not stripped.*/\1/p'`; do
-	$STRIP -g "$f" || :
-done
+find "$RPM_BUILD_ROOT" -type f \( -perm -0100 -or -perm -0010 -or -perm -0001 \) | grep -v "^${RPM_BUILD_ROOT}/\?usr/lib/debug"  | \
+    xargs -r -P$NCPUS -n32 sh -c "file \"\$@\" | grep -v ' shared object,' | sed -n -e 's/^\(.*\):[ 	]*ELF.*, not stripped.*/\1/p' | xargs -I\{\} $STRIP -g \{\}" ARG0
```

Comment:
what about doing a bit less of escaping?

`sh -c 'file "$@" | grep -v " shared object," | sed -n -e "s/^\(.*\):[ \t]*ELF.*, not stripped.*/\1p" | xargs $STRIP -g' ARG0`

Reply:
All these xargs-conversions will need to use find -print0 | xargs -0, see https://bugzilla.redhat.com/show_bug.cgi?id=1721348#c4